compose/bin/setup

```diff
@@ -3,14 +3,10 @@ set -o errexit
 source env/db.env
 BASE_URL=${1:-magento2.test}
 
-ES_HOST=elasticsearch
-ES_PORT=9200
```

Comment:
Need to add these back in

Reply:
I think this is where we need updates. These settings are file, but need to pull in from envvar if it exists.

I didn't test but something like this pseudo code may work:

```
ES_HOST="${ES_HOST:=elasticsearch}"
ES_PORT="${ES_PORT:=9200}"
```

This way we don't need to define anything in the XML files.

Reply:
On second thought, maybe be better to create an `env/elasticsearch.env` file for these similar to https://github.com/markshust/docker-magento/blob/master/compose/env/db.env

Reply:
@markshust I think this push request (commit) should only keep changes about `HEALTHCHECK`.
For other environmental variables, I will create another one for it.